crates/core/tedge_actors/src/messages.rs

```diff
@@ -0,0 +1,7 @@
+use std::fmt::Debug;
+
+/// A message exchanged between two actors
+pub trait Message: Debug + Send + Sync + 'static {}
+
+/// There is no need to tag messages as such
```

Comment:
Didn't understand that comment or the relevance of that impl block.

Reply:
Otherwise an empty implementation will have to be provided for all message types.

Reply:
There is no need for a `#[derive(Message)]` macro thanks to this `impl` block that tells that anything that is `Debug + Send + Sync + 'static` is a `Message`.

Reply:
We agreed to make Message implicit as long as the constraints are okay.

Pros: a String or u64 is a message type.